frontend/models/product-list/index.ts

```diff
@@ -227,6 +242,31 @@ type ApiProductList = NonNullable<
    >[0]['attributes']
 >;
 
+function convertAncestorsToStrapiFormat(
+   ancestors: any
+): ApiProductList['parent'] | null {
+   const ancestor = ancestors.shift();
+   if (ancestor == null) {
+      return null;
+   }
+   if (ancestor.wikiid === 11000) {
```

Comment:
Any reason to check wikiid rather than title === 'Root'?

Reply:
i thought that `wikiid` would be the most stable identifier but I asked about this in my CR section.